client/dashboard/domains/name-servers/upsell-nudge.tsx

```diff
@@ -9,15 +9,16 @@ import illustrationUrl from './upsell-nudge-illustration.svg';
 
 interface Props {
 	domainName: string;
+	domainSiteSlug: string;
 }
 
-export default function UpsaleNudge( { domainName }: Props ) {
+export default function UpsaleNudge( { domainName, domainSiteSlug }: Props ) {
```

Comment:
Is this `Upsale` or `Upsell` 😄

Reply:
Oops, good catch. :)
I'll update it right away.